tests/e2e-kernel-test.sh

```diff
@@ -76,26 +76,16 @@ for TEST in $TESTS; do
     info
 
     rm -f $SCRIPT_TMP_DIR/build-$$
-    rm -f $SCRIPT_TMP_DIR/ebpf-$$
 
-    events=$(./dist/tracee-rules --rules $TEST --list-events)
-
-    ./dist/tracee-ebpf \
+    ./dist/tracee \
         --install-path $TRACEE_TMP_DIR \
         --cache cache-type=mem \
         --cache mem-cache-size=512 \
-        --output format:gob \
-        --output option:parse-arguments \
-        --scope container=new \
-        --events $events \
-        2>$SCRIPT_TMP_DIR/ebpf-$$ |
-        ./dist/tracee-rules \
-            --input-tracee=file:stdin \
-            --input-tracee format:gob \
-            --rules $TEST 2>&1 |
+        --output json \
+        --scope container=new 2>&1 |
```

Comment:
```suggestion
        --scope container=new 2>&1 \
        | tee "$SCRIPT_TMP_DIR/build-$$" &
```

Added backslash at the end of the line and moved the pipe to the beginning of the next. This makes the intent more clear.

Removed the second `2>&1` after tee since redirecting both stdout and stderr is not necessary; tee already captures both streams.

WDYT?